### PR TITLE
Feature/filter badges by letter

### PR DESCRIPTION
**Pull request by zalapeach**
# This pull request involves
- Adds filters by letter in badges_engine/badges#index page
- Adds specs

### CODE REVIEW

**Comment by vicmaster** [comment]
LGTM

File: vendor/engines/badges_engine/app/controllers/badges_engine/badges_controller.rb [hunk 1]
```diff
@@ -8,6 +8,8 @@ class BadgesController < ApplicationController
     def index
       if params[:search]
         @badges = Badge.search_badges(params[:search]).sort_by_column_direction
+      elsif params[:letter]
+        @badges = Badge.by_letter(params[:letter])
```
Comment (ingedmundo) [inline vendor/engines/badges_engine/app/controllers/badges_engine/badges_controller.rb]: wouldn't be more descriptive 'search_by_letter' ?

**Comment by ingedmundo** [comment]
LGTM

File: vendor/engines/badges_engine/app/controllers/badges_engine/badges_controller.rb (hunk 1 above)
Comment (vicmaster) [inline vendor/engines/badges_engine/app/controllers/badges_engine/badges_controller.rb]: Indeed

**Pull request merged**